README.md

```diff
@@ -986,6 +987,91 @@ your application.
   of [sidekiq](https://github.com/mperham/sidekiq) gem.
 <sup>[[link](#background-email)]</sup>
 
+
+## Active Support Core Extensions
+
+* <a name="try-bang"></a>
+  Prefer Ruby 2.3.0's safe navigation operator `&.` over `ActiveSupport#try!`.
+<sup>[[link](#try-bang)]</sup>
+
+```ruby
+# bad
+nil.try! :fly
+
+# good
+nil&.fly
+```
+
+* <a name="string_starts_ends_with"></a>
+  Prefer Ruby's `String#start_with?` and `String#end_with?` over `ActiveSupport`
+  aliases `String#starts_with?` and `String#ends_with?`
+<sup>[[link](#string_starts_ends_with)]</sup>
+
+```ruby
+# bad
+'the day'.starts_with? 'th'
+'the day'.ends_with? 'ay'
+
+# good
+'the day'.start_with? 'th'
+'the day'.end_with? 'ay'
+```
+
+* <a name="object_inclusion"></a>
+  Prefer Ruby's `String#include?` and `Array#include?` over ActiveSupport's `Object#in?`
+<sup>[[link](#object_inclusion)]</sup>
+
+```ruby
+# bad
+1.in? [1, 2]
+'day'.in? 'the day'
+
+# good
+[1, 2].include? 1
+'the day'.include? 'day'
+```
+
+* <a name="array_forty_two"></a>
+  Prefer Ruby's `Array#[]` over ActiveSupport's `Array#forty_two?`
+<sup>[[link](#array_forty_two)]</sup>
+
+```ruby
+# bad
+(1..50).to_a.forty_two
+
+# good
+(1..50).to_a[41]
+```
+
+* <a name="inquiry"></a>
+  Prefer Ruby's comparison operators over ActiveSupport's `Array#inquiry`, `Numeric#inquiry' and `String#inquiry`
+<sup>[[link](#inquiry)]</sup>
+
+```ruby
+# bad - String#inquiry
+ruby = 'two'.inquiry
+ruby.two?
+
+# good
+ruby = 'two'
+ruby == 'two'
+
+# bad - Array#inquiry
+pets = %w(cat dog).inquiry
+pets.gopher?
+# good
+pets.include? 'cat'
```

Comment:
It feels self explanatory, but you didn't setup the `pets` object for your good example and it's still using the `inquiry` version.